frontend/src/pages/Nodes/NodesTable.js

```diff
@@ -49,28 +49,28 @@ const NodesTable = ({ nodes, classes }) => {
           {strings.nodesDashboard.network}
         </Typography>
       </div>
```

Comment:
use right formatter/extensions for vs code more details can be found in the contributor guide